.yarn/patches/@metamask-assets-controllers-npm-45.1.0-d914c453f0.patch

```diff
@@ -1,59 +0,0 @@
-diff --git a/dist/assetsUtil.cjs b/dist/assetsUtil.cjs
-index 48571b8c1b78e94d88e1837e986b5f8735ac651b..61246f51500c8cab48f18296a73629fb73454caa 100644
---- a/dist/assetsUtil.cjs
-+++ b/dist/assetsUtil.cjs
-@@ -3,6 +3,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
-     return (mod && mod.__esModule) ? mod : { "default": mod };
- };
- Object.defineProperty(exports, "__esModule", { value: true });
-+function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { newObj[key] = obj[key]; } } } newObj.default = obj; return newObj; } }
- exports.fetchTokenContractExchangeRates = exports.reduceInBatchesSerially = exports.divideIntoBatches = exports.ethersBigNumberToBN = exports.addUrlProtocolPrefix = exports.getFormattedIpfsUrl = exports.getIpfsCIDv1AndPath = exports.removeIpfsProtocolPrefix = exports.isTokenListSupportedForNetwork = exports.isTokenDetectionSupportedForNetwork = exports.SupportedStakedBalanceNetworks = exports.SupportedTokenDetectionNetworks = exports.formatIconUrlWithProxy = exports.formatAggregatorNames = exports.hasNewCollectionFields = exports.compareNftMetadata = exports.TOKEN_PRICES_BATCH_SIZE = void 0;
- const controller_utils_1 = require("@metamask/controller-utils");
- const utils_1 = require("@metamask/utils");
-@@ -233,7 +234,7 @@ async function getIpfsCIDv1AndPath(ipfsUrl) {
-     const index = url.indexOf('/');
-     const cid = index !== -1 ? url.substring(0, index) : url;
-     const path = index !== -1 ? url.substring(index) : undefined;
--    const { CID } = await import("multiformats");
-+    const { CID } = _interopRequireWildcard(require("multiformats"));
-     // We want to ensure that the CID is v1 (https://docs.ipfs.io/concepts/content-addressing/#identifier-formats)
-     // because most cid v0s appear to be incompatible with IPFS subdomains
-     return {
-diff --git a/dist/token-prices-service/codefi-v2.mjs b/dist/token-prices-service/codefi-v2.mjs
-index a13403446a2376d4d905a9ef733941798da89c88..3c8229f9ea40f4c1ee760a22884e1066dac82ec7 100644
---- a/dist/token-prices-service/codefi-v2.mjs
-+++ b/dist/token-prices-service/codefi-v2.mjs
-@@ -12,8 +12,7 @@ var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (
- var _CodefiTokenPricesServiceV2_tokenPricePolicy;
- import { handleFetch } from "@metamask/controller-utils";
- import { hexToNumber } from "@metamask/utils";
--import $cockatiel from "cockatiel";
--const { circuitBreaker, ConsecutiveBreaker, ExponentialBackoff, handleAll, retry, wrap, CircuitState } = $cockatiel;
-+import { circuitBreaker, ConsecutiveBreaker, ExponentialBackoff, handleAll, retry, wrap, CircuitState } from "cockatiel";
```

Comment:
I think removing this part of the patch is what cause `__import__ is not a function` err: here is the initial PR where i've introduced this https://github.com/MetaMask/metamask-extension/pull/27629#discussion_r1791797810

Reply:
thank you @sahar-fehri!